The `-mtune=native` flag is not supported on aarch64, remove it.

--- CMakeLists.txt.orig	2018-07-11 20:29:53 UTC
+++ CMakeLists.txt
@@ -182,11 +182,11 @@ if (NOT MSVC)
     add_compile_options( -pthread )

     add_compile_options("$<$<CONFIG:RELWITHDEBINFO>:-O2>")
-    add_compile_options("$<$<CONFIG:RELWITHDEBINFO>:-mtune=native>")
+    #add_compile_options("$<$<CONFIG:RELWITHDEBINFO>:-mtune=native>")

     add_compile_options("$<$<CONFIG:RELEASE>:-O2>")
     add_compile_options("$<$<CONFIG:RELEASE>:-g0>")
-    add_compile_options("$<$<CONFIG:RELEASE>:-mtune=native>")
+    #add_compile_options("$<$<CONFIG:RELEASE>:-mtune=native>")

     add_compile_options("$<$<CONFIG:DEBUG>:-O0>")
